recipes/kaiju/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: ee6c319ab3e58db846139810705bfe33306618d78d75d715fac623f302be9e72
 
 build:
-  number: 0  
+  number: 1
```

Comment:
Instead of the build number increase, can you update the package to version `1.6.2` and update the `sha256` accordingly?